.python-version

```diff
@@ -0,0 +1 @@
+3.13
```

Comment:
As far as I know, there isn't anything 3.13 specific that we're using. In fact, the GitHub Actions runner uses 3.12. I'd remove this file since the repository should be compatible with a few older versions of Python as well.